CHANGELOG.md

```diff
@@ -1,10 +1,11 @@
-## 0.15.10
+## 0.15.11
 
 ### Enhancements
 
 * **Enhance `pdfminer` element cleanup** Expand removal of `pdfminer` elements to include those inside all `non-pdfminer` elements, not just `tables`.
```

Comment:
Pretty sure these bullets were in the `0.15.10` release. If so, may need a fresh section for `0.15.11`

Reply:
Those look accurate based on the release notes: https://github.com/Unstructured-IO/unstructured/releases/tag/0.15.10

Reply:
Looks like the `0.15.10` section is missing on the branch and `0.15.10` was released already. Think your new entry needs to be in a fresh `0.15.11` section.

<img width="1089" alt="image" src="https://github.com/user-attachments/assets/eb7591e4-84ad-4bdb-9ac3-36c63b4d11db">

Reply:
Fixed